exporter/kafkaexporter/kafka_exporter_test.go

```diff
@@ -101,6 +101,8 @@ func TestNewExporter_err_auth_type(t *testing.T) {
 		Metadata: Metadata{
 			Full: false,
 		},
+		CompressionCodec: defaultCompression,
+		CompressionLevel: defaultCompressionLevel,
```

Comment:
Are these needed in this test? I couldn't find where it would change anything being tested here?

Reply:
Yes, this test will fail if not.